app/controllers/admin/dashboard_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Admin::DashboardController < Admin::AdminController
+  def dashboard
+    authorize Project
```

Comment:
Project will be used by 3 out of the 4 features that are a part of the dashboard. But I can remove this until it is needed, if you think that is best.